app/scripts/controllers/client/ViewClientController.js

```diff
@@ -48,6 +48,26 @@
                     }
                 });
 
+                scope.clickEvent = function (eventName, accountId, savingProductId) {
```

Comment:
would be better to have meaningful names:

Method name might be : navigateToSavingsOrDepositAccount
The last parameter passed is productType (not savingProductId)